usaspending_api/api_contracts/contracts/v2/references/filter_tree/psc/group.md

```diff
@@ -0,0 +1,73 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# PSC
+
+This endpoint is used to power USAspending.gov's "Product or Service Code" search component on the advanced search page.
+The response is a forest of search filter nodes, which despite having a unified structure represent different
+database fields based on depth in the tree.
+
+## Toptier Search [GET /api/v2/references/filter_tree/psc/{group}/{?depth}{?filter}]
+
+Returns PSCs under the provided group
++ Request (application/json)
+    + Parameters
+        + `group`: `Product` (required, string)
+        Parent group of PSCs to return
```

Comment:
Since this parameter will allow any string and simply produce no results, can you list the three group strings which have the potential to return data in the contracts?